src/chem-tests/nitrate.ts

```diff
@@ -0,0 +1,99 @@
+import { ChemistryTest, ChemTestRatingType, ChemTestType, StepType } from "../utils/chem-types";
+import t from "../utils/translation/translate";
+
+import nitrate_0 from "../assets/chemistry/nitrate/nitrate-0.svg";
+import nitrate_1A from "../assets/chemistry/nitrate/nitrate-1a.svg";
+import nitrate_1B from "../assets/chemistry/nitrate/nitrate-1b.svg";
+import nitrate_2A from "../assets/chemistry/nitrate/nitrate-2a.svg";
+import nitrate_2B from "../assets/chemistry/nitrate/nitrate-2b.svg";
+import nitrate_2C from "../assets/chemistry/nitrate/nitrate-2c.svg";
+import nitrate_2D from "../assets/chemistry/nitrate/nitrate-2d.svg";
+import nitrate_2E from "../assets/chemistry/nitrate/nitrate-2e.svg";
+import nitrate_2F from "../assets/chemistry/nitrate/nitrate-2f.svg";
+import nitrate_2G from "../assets/chemistry/nitrate/nitrate-2g.svg";
+import nitrate_2H from "../assets/chemistry/nitrate/nitrate-2h.svg";
+import nitrate_2I from "../assets/chemistry/nitrate/nitrate-2i.svg";
+import nitrate_2J from "../assets/chemistry/nitrate/nitrate-2j.svg";
+import nitrate_2K from "../assets/chemistry/nitrate/nitrate-2k.svg";
+import nitrate_2L from "../assets/chemistry/nitrate/nitrate-2l.svg";
+import nitrate_2M from "../assets/chemistry/nitrate/nitrate-2m.svg";
+import nitrate_2N from "../assets/chemistry/nitrate/nitrate-2n.svg";
+import nitrate_2O from "../assets/chemistry/nitrate/nitrate-2o.svg";
+import nitrate_2P from "../assets/chemistry/nitrate/nitrate-2p.svg";
+import nitrate_2Q from "../assets/chemistry/nitrate/nitrate-2q.svg";
+import nitrate_2R from "../assets/chemistry/nitrate/nitrate-2r.svg";
+import nitrate_2S from "../assets/chemistry/nitrate/nitrate-2s.svg";
+import nitrate_2T from "../assets/chemistry/nitrate/nitrate-2t.svg";
+import nitrate_2U from "../assets/chemistry/nitrate/nitrate-2u.svg";
+import nitrate_2V from "../assets/chemistry/nitrate/nitrate-2v.svg";
+import nitrate_2W from "../assets/chemistry/nitrate/nitrate-2w.svg";
+import nitrate_2X from "../assets/chemistry/nitrate/nitrate-2x.svg";
+import nitrate_3A_0 from "../assets/chemistry/nitrate/nitrate-3a-0.svg";
+import nitrate_3A_5 from "../assets/chemistry/nitrate/nitrate-3a-5.svg";
+import nitrate_3A_20 from "../assets/chemistry/nitrate/nitrate-3a-20.svg";
+import nitrate_3B_0 from "../assets/chemistry/nitrate/nitrate-3b-0.svg";
+import nitrate_3B_5 from "../assets/chemistry/nitrate/nitrate-3b-5.svg";
+import nitrate_3B_20 from "../assets/chemistry/nitrate/nitrate-3b-20.svg";
+import nitrate_3C_0 from "../assets/chemistry/nitrate/nitrate-3c-0.svg";
+import nitrate_3C_5 from "../assets/chemistry/nitrate/nitrate-3c-5.svg";
+import nitrate_3C_20 from "../assets/chemistry/nitrate/nitrate-3c-20.svg";
+
+export const nitrateTest: ChemistryTest = {
+  type: ChemTestType.nitrate, label: t("CHEM.NITRATE.TEST"), units: t("CHEM.NITRATE.UNIT"),
```

Comment:
Need to add `CHEM.NITRATE.UNIT` to the translation files.

Reply:
Need this in both translation JSON files:
"CHEM.NITRATE.UNIT": "ppm",

Reply:
This wasn't visible before because the nitrate test had the wrong unit:
```
type: ChemTestType.nitrate, label: t("CHEM.NITRATE.TEST"), units: t("CHEM.AIR.UNIT")
```